.github/CODEOWNERS

```diff
@@ -1,5 +1,31 @@
+# Firefox Translations review group
+.dockerignore @mozilla/firefox-translations
+.github @mozilla/firefox-translations
+.gitignore @mozilla/firefox-translations
+.gitmodules @mozilla/firefox-translations
+docker @mozilla/firefox-translations
+docs @mozilla/firefox-translations
+utils @mozilla/firefox-translations
+CODE_OF_CONDUCT.md @mozilla/firefox-translations
+LICENSE @mozilla/firefox-translations
+poetry.lock @mozilla/firefox-translations
+pyproject.toml @mozilla/firefox-translations
+README.md @mozilla/firefox-translations
+Taskfile.yml @mozilla/firefox-translations
+
+# Translations Training review group
+configs @mozilla/translations-training
+pipeline @mozilla/translations-training
+snakemake @mozilla/translations-training
+tests @mozilla/translations-training
+tracking @mozilla/translations-training
+
+# Translations Inference review group
+inference-engine @mozilla/translations-inference
```

Comment:
Can we call it just "inference" to make it shorter?

Reply:
Sounds good to me.